packages/block-editor/src/utils/transform-styles/index.js

```diff
@@ -23,7 +18,7 @@ import wrap from './transforms/wrap';
  * @return {Array} converted rules.
  */
 const transformStyles = ( styles, wrapperClassName = '' ) => {
-	return map( styles, ( { css, baseURL } ) => {
+	return Object.values( styles ?? [] ).map( ( { css, baseURL } ) => {
```

Comment:
I'm curious, why did you go for `[]` instead of `{}`? They both work fine, in hindsight, I just found it surprising at first.

Reply:
Because various usages of `transformStyles()` across the codebase work with either arrays or objects of styles, and I had to pick one of them 😉 Since we're doing a transformation to an array anyway, it felt better to just default to an array, regardless of the fact that we're doing an `Object.values()` on it. Also, the function is already documented to expect an array of styles, so we're matching that. Regardless, I think the fact that it works with both objects and arrays is a bit vague and we could definitely improve that in the future.